javascript/input-and-string/js-inputs-and-strings_input-to-uppercase/js/index.js

```diff
@@ -10,3 +10,12 @@ Convert the input value to uppercase when clicking the button
 Hint: Access the value by using `.value` on the input element
 
 */
+const firstInput = document.querySelector('[data-js="first-input"]');
+const button = document.querySelector('[data-js="button-uppercase"]');
+const output = document.getElementById("help-text");
```

Comment:
Du brauchst keinen Zugriff auf dieses Element, wo nur steht: "Type something to test your code."